### ISSUE TITLE
Where is the jacobian implementation for the backpropogation?

### ISSUE THREAD

**Issue by canxkoz** [pals-ttic/sjc#22]
I could not find the jacobian implementation for the backpropagation. I need to use the jacobian loss function which is shared in the paper, however I cannot find it. Could anyone please help me find where the loss function is implemented? 
![image_6487327](https://github.com/pals-ttic/sjc/assets/35275446/92e2685b-2006-4e70-84e8-eeabf272baeb)
Above is a screenshot taken from the paper. I am searching for the implemetation of the chain rule in code form.
Could you please help me find where that specific function is? 
Thank you for your time,
